website/pages/gamejam_en.md

```diff
@@ -1,39 +1,13 @@
 <img src="/static/img/logga.png" alt="LiU Game Jam" id="gamejam-logo">
-<h1>Global Game Jam 2021</h1>
-Get ready for our upcoming event, Global Game Jam!
-This year's edition of Global Game Jam will be held 29th - 31st January.
-Because COVID-19 is still wreaking havoc in the world, the jam will be held remotely in our [Discord server](https://discord.gg/tP2kDvgQKn). Also check out our [Facebook event](https://fb.me/e/1SCmIAaRH).
-Join our [Itch.io](https://itch.io/jam/global-game-jam-linkoping-2021)!
+<h1>Syltprovning</h1>
+Get ready for our upcoming event, Syltprovning!
+We will for the first time ever the 20th April 5:15pm arrange a Syltprovning! This event is aimed at you who are curious about game jams, but are unsure where to begin.
+It is also an opportunity for experienced jammers to share their experiences and to be nostalgic about jams that have been.
 
-**To participate**
+The event will be coposed of a short presentation where we present what a jam is, how our jams work, and some anecdotes old jammers.
+This is followed by some playtesting of old game jam games, chill and talking about game jams, games, and all sorts of things.
```

Comment:
```suggestion
The event will be composed of a short presentation where we present what a jam is, how our jams work, and some anecdotes from old jammers.
This will be followed by some play testing of old game jam games while chilling and talking about game jams, games, and all sorts of things.
```

Reply:
Alternativt "be composed" -> "consist".